packages/eslint-plugin/src/rules/ban-types.ts

```diff
@@ -20,6 +16,31 @@ type Options = [
 ];
 type MessageIds = 'bannedTypeMessage';
 
+function stringifyTypeName(
+  node: TSESTree.EntityName,
+  sourceCode: TSESLint.SourceCode,
+): string {
+  return sourceCode.getText(node).replace(/ /g, '');
```

Comment:
I'm not sure I understand what the `replace(...)` is for?
A `TypeReference` shouldn't ever have spaces in it?